### PR TITLE
CIV-4599: Defendant Response - Claimant LR email notifications

### PR DESCRIPTION
**Pull request by kenneth-hmcts**
**Before creating a pull request make sure that:**

- [x] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [x] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link ###
https://tools.hmcts.net/jira/browse/CIV-4599

### Change description ###
This PR contains that changes for the translation from citizen UI part to ccd definition part

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/common/form/models/yesNo.ts [hunk 1]
```diff
@@ -11,3 +11,7 @@ export enum YesNoNotReceived {
   NO = 'no',
   NOT_RECEIVED = 'not-received'
 }
+export enum YesNoToCcdTranslation {
```
Comment (laarmada) [inline src/main/common/form/models/yesNo.ts:14]: for generic purpose we can use YesNoUpperCamelCase instead of YesNoToCcdTranslation

**Review by laarmada — CHANGES_REQUESTED** [review]

File: src/main/common/models/ccdResponse/ccdAgreedMediation.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import {Mediation} from '../../models/mediation/mediation';
+import {YesNoToCcdTranslation} from '../../../common/form/models/yesNo';
+
+export const toAgreedMediation = (mediation: Mediation): string => {
+  return mediation?.canWeUse?.option ? YesNoToCcdTranslation.YES
```
Comment (commitBlob) [inline src/main/common/models/ccdResponse/ccdAgreedMediation.ts:5]: I like ternary operators but Sonar is complaining that nested ternaries are code smell.
https://sonarcloud.io/project/issues?resolved=false&sinceLeakPeriod=true&types=CODE_SMELL&pullRequest=880&id=civil-citizen-ui&open=AYN6VxBIdMmx1aShV3tL

Would you mind refactoring it into if/else  conditions please

File: src/main/common/form/models/yesNo.ts (hunk 1 above)
Comment (kenneth-hmcts) [reply src/main/common/form/models/yesNo.ts]: Changed to YesNoUpperCamelCase in the model

File: src/main/common/models/ccdResponse/ccdAgreedMediation.ts (hunk 2 above)
Comment (kenneth-hmcts) [reply src/main/common/models/ccdResponse/ccdAgreedMediation.ts]: Code is refactored and the code smell should be removed
https://sonarcloud.io/project/issues?resolved=false&types=CODE_SMELL&sinceLeakPeriod=true&pullRequest=880&id=civil-citizen-ui

**Review by commitBlob — APPROVED** [review]

**Pull request merged**